grl/environment/memory_lib.py

```diff
@@ -6,7 +7,22 @@
 Dimensions: AxZxMxM
 """
 
-mem_0 = np.array([
+def get_memory(memory_id: int, n_obs: int, n_actions: int, n_mem_states: int = 2) -> np.ndarray:
+    current_module = globals()
```

Comment:
Huh. Interesting. Never heard of `globals()`

Reply:
Was trying to find a way to search through all the global names within the `memory_lib` module. Open to alternatives too!

Reply:
This is fine for now. It seems like maybe it's an indication that we should find a better way of organizing things though.